cmd/dagger/cmd/project/templates/hello.cue

```diff
@@ -0,0 +1,30 @@
+// The first time you run the hello
+// action as `dagger do hello --log-format plain`,
+// make sure to run `dagger project update` first,
+// so that all required dependencies are available.
+
+package hello
+
+import (
+	"dagger.io/dagger"
+	"universe.dagger.io/bash"
+	"universe.dagger.io/alpine"
+
+)
+
+dagger.#Plan & {
+	actions: {
+		_alpine: alpine.#Build & {
+			packages: {
```

Comment:
Could be directory: `packages: bash: _`.

Btw I've seen a lot of user using that pattern for running bash, I think we should improve `bash.#Run` so there is a default `bash` installed if user do not provide his image.
/cc @helderco

Reply:
Ugh ... this is my bad, I think I broke the CUE linter when converting it to dagger (`dagger do lint cue`).

The linter should have caught this:

```
$ make cuefmt
$ git diff
diff --git a/cmd/dagger/cmd/project/templates/hello.cue b/cmd/dagger/cmd/project/templates/hello.cue
index f0b45533..5d3fdab1 100644
--- a/cmd/dagger/cmd/project/templates/hello.cue
+++ b/cmd/dagger/cmd/project/templates/hello.cue
@@ -15,9 +15,7 @@ import (
 dagger.#Plan & {
        actions: {
                _alpine: alpine.#Build & {
-                       packages: {
-                               bash: _
-                       }
+                       packages: bash: _
                }
```

Reply:
Fixed the linter: https://github.com/dagger/dagger/pull/2159

@marcosnils Can you rebase this PR after #2159 is merged? Should surface CUE linting errors

Reply:
done!